microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -19,15 +20,18 @@
     description="""
     This test suite check DPDK functionality
     """,
+    requirement=simple_requirement(
+        supported_features=[Sriov],
+        min_core_count=8,
+        min_nic_count=2,
+        supported_os=[Ubuntu, Redhat],
```

Comment:
what's the reason to limit the distro? It blocks the test cases to run in more distros.

Reply:
Those are the only ones that are implemented at the moment, I can remove it, it will just end up skipping the tests for other distros

Reply:
If it's just "we" are not implemented, it shouldn't be added here. We can implement it later, when it runs against other distros. If it's "the feature (DPDK)" is not implemented on other distros, they need to be added here.

Reply:
Well, the issue still exists that adding those as a requirement breaks running images with those OSs. You can repro by making a nop test and adding the ubuntu or redhat requirement and trying to run it with either image.

I'll remove it and I'd like to accept this PR without the OS requirement since it's not... required 😄